core/math/static_raycaster.h

```diff
@@ -80,20 +80,20 @@ class StaticRaycaster : public RefCounted {
 
 	public:
 		Vector3 org; //!< Ray origin + tnear
-		float tnear; //!< Start of ray segment
+		float tnear = 0; //!< Start of ray segment
```

Comment:
Floats should be initialized with float literals `0.0f`.

You should also sync these values with what is actually being done in the constructor right above. The intializations from the constructor with constant expressions can be moved to the declarations.